HrBarChart.tsx

```
@@ -0,0 +1,340 @@
+            fill={colors.primary}
+            radius={[0, 4, 4, 0]}
+            barSize={20}
+            label={<CustomBarLabel />}
+            animationDuration={1000}
+            animationBegin={0}
+          />
+        </BarChart>
+      </ResponsiveContainer>
+    </Box>
+  );
+};import { Box, Flex, Text } from '@chakra-ui/react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Box'. Consider to delete it or rename it.

'Box' is defined here:


(lint/suspicious/noRedeclare)